lib/csu/mips64c128/Makefile

```diff
@@ -1,63 +1,35 @@
-#
 # $FreeBSD$
-#
+
+.PATH:	${.CURDIR}/../common-cheri ${.CURDIR}/../common
+
+OBJS=		Scrt1.o crt1.o
+CFLAGS+=	-I${.CURDIR} -I${.CURDIR:H}/common \
+		-I${.CURDIR:H}/common-cheri \
+		-I${SRCTOP}/lib/libc/include
+CFLAGS+=	-mxgot -mllvm -mxmxgot
+CFLAGS+=	-DCRT_IRELOC_SUPPRESS
+# Purecap code requires crt_init_globals to be called before using jump tables
+CFLAGS+=	-fno-jump-tables
+
+FILES=		${OBJS}
+FILESMODE=      ${LIBMODE}
```

Comment:
it might be with fixing the use of spaces in this area

Reply:
It's all going to go away when we merge in the commit that moves duplicated lines to ../Makefile.inc.  I do need to get CI green though.